instancing.js

```diff
@@ -558,14 +558,17 @@ export class InstancedGeometryAllocator {
           name,
           Type,
           itemSize,
+          itemCount
         } = spec;
 
         // compute the minimum size of a texture that can hold the data
         let neededItems4 = numGeometries * maxDrawCallsPerGeometry * maxInstancesPerDrawCall;
+        if (itemCount)
+          neededItems4 = itemCount;

```

Comment:
Can we clean up this api? This seems like derived data and should not be overridden in this way.

Reply:
Cleaned up. https://github.com/webaverse/app/pull/3313/commits/16dcea6ff45eabf9c1b55dade26ce1286d419860

Reply:
Another cleanup: https://github.com/webaverse/app/pull/3313/commits/f0f396a1438e67ac8971986fd78643eaa023ee41